TESTOWNERS

```diff
@@ -331,10 +331,10 @@
 # tool_host_cross_arch_tests @andrewkolos @flutter/tool
 # tool_integration_tests @christopherfujino @flutter/tool
 # tool_tests @andrewkolos @flutter/tool
-# verify_binaries_codesigned @xilaizhang @flutter/releases
+# verify_binaries_codesigned @cbracken @flutter/releases
 # web_canvaskit_tests @yjbanov @flutter/web
 # web_integration_tests @yjbanov @flutter/web
 # web_long_running_tests @yjbanov @flutter/web
 # web_tests @yjbanov @flutter/web
 # web_skwasm_tests @eyebrowsoffire @flutter/web
-# web_tool_tests @eliasyishak @flutter/tool
+# web_tool_tests @yjbanov @flutter/tool
```

Comment:
@yjbanov if you're not the right owner for this, can you suggest an owner? @eyebrowsoffire maybe?